<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorForm</class>
 <widget class="QWidget" name="MapEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widgetButtons" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <widget class="QPushButton" name="createItem">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Item</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createTurf">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Turf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="newMap">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>New map</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="posxEdit">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="posyEdit">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="poszEdit">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>21</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="xyz_label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>60</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>  X         Y       Z</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeItem">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Item</string>
        </property>
       </widget>
       <widget class="QPushButton" name="beginSelection">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Selection</string>
        </property>
       </widget>
       <widget class="QLabel" name="cursor_label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="resizeMap">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Resize Map</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveMapJson">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Json</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadMapJson">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Json</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QListWidget" name="listWidget">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
           <widget class="QListWidget" name="listWidgetTurf"/>
          </widget>
          <widget class="QGraphicsView" name="graphicsView"/>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QListWidget" name="listWidgetTile"/>
           <widget class="QListWidget" name="listWidgetVariables"/>
           <widget class="QTabWidget" name="tabWidgetVariables">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="tabRaw">
             <attribute name="title">
              <string>Raw</string>
             </attribute>
             <widget class="QLineEdit" name="lineEditRaw">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>113</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tabString">
             <attribute name="title">
              <string>As String</string>
             </attribute>
             <widget class="QLineEdit" name="lineEditAsString">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>113</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tabInt">
             <attribute name="title">
              <string>As Int</string>
             </attribute>
             <widget class="QLineEdit" name="lineEditAsInt">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>113</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>As Bool</string>
             </attribute>
             <widget class="QLineEdit" name="lineEditAsBool">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>113</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
